frontend/apps/filemanager/filemanager.lua

```diff
@@ -1201,4 +1178,70 @@ function FileManager:onRefreshContent()
     self:onRefresh()
 end
 
+function FileManager:onShowFolderMenu()
+    local button_dialog
+    local function genButton(button_text, button_path)
+        return {{
+            text = button_text,
+            align = "left",
+            font_face = "smallinfofont",
+            font_size = 22,
+            font_bold = false,
+            avoid_text_truncation = false,
```

Comment:
(Was going to ask "why force that?!" - but testing without, the wrap is ugly with the second line going under the indentation & symbol - and it would be too hard to make nice - so, ok with that :)